src/admin-views/aggregator/origins/ics.php

```diff
@@ -42,8 +42,8 @@ class="tribe-ea-field tribe-ea-media_button tribe-dependent button button-second
 		<button type="submit" class="button button-primary tribe-preview">
 			<?php esc_html_e( 'Preview', 'the-events-calendar' ); ?>
 		</button>
-		<button type="button" class="button tribe-clear-filters">
-			<?php esc_html_e( 'Clear Filters', 'the-events-calendar' ); ?>
+		<button type="cancel" class="button tribe-cancel">
```

Comment:
I matched this: https://github.com/moderntribe/the-events-calendar/blob/develop/src/admin-views/aggregator/tabs/import-form.php#L177

Behavior is identical to that button.